velox/docs/functions/spark/aggregate.rst

```diff
@@ -24,7 +24,6 @@ General Aggregate Functions
 .. spark:function:: bloom_filter_agg(hash, estimatedNumItems, numBits) -> varbinary
 
     Creates bloom filter from input hashes and returns it serialized into VARBINARY.
-    The caller is expected to apply xxhash64 function to input data before calling bloom_filter_agg.
```

Comment:
Why did you remove this sentence? Is it incorrect?

At the same time, I see that "``hash`` cannot be null." is still part of the docs. Please, take time to carefully review the documentation and make sure it is accurate and complete.

Reply:
The `bloom_filter_agg` can support `null` values and is no longer expected to apply the `xxhash64` function to input data before calling `bloom_filter_agg`. I have updated the docs.